src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ApplicationPropertyPage.xaml

```diff
@@ -43,6 +43,11 @@
       <DataSource Persistence="ProjectFileWithInterception"
                   HasConfigurationCondition="False" />
     </BoolProperty.DataSource>
+    <BoolProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>false</NameValuePair.Value>
+      </NameValuePair>
+    </BoolProperty.Metadata>
```

Comment:
Does it work to just set `Visible="False"` on the `BoolProperty` directly instead? That might be simpler. I haven't tested that scenario however.

Reply:
Doesn't work. You get this error:
```
Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.EvaluationException: Conditional visibility expression references unknown property Application::TargetMultipleFrameworks
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.VisibilityConditionEvaluator.<>c__DisplayClass2_0.<.ctor>g__GetProperty|1(String pageName, String propertyName)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.VisibilityConditionEvaluator.<>c__DisplayClass2_0.<.ctor>g__GetUnconfiguredPropertyValue|2(String pageName, String propertyName)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.VisibilityConditionEvaluator.<>c__DisplayClass2_0.<.ctor>g__GetEvaluatedValue|4(String pageName, String propertyName)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.Function`3.TryInvoke(Object[] args, Object& result)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.EvaluationEnvironment.<Evaluate>g__Eval|2_1(Object node, <>c__DisplayClass2_0& )
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.EvaluationEnvironment.<Evaluate>g__Eval|2_1(Object node, <>c__DisplayClass2_0& )
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.EvaluationEnvironment.Evaluate(String expression)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.Expressions.VisibilityConditionEvaluator.UpdateConditionalVisibilities()
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.PropertyContext.<SetProperties>g__Initialize|21_0(Property property)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.PropertyContext.SetProperties(ImmutableArray`1 properties)
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.PropertyDataAccess.<LoadPropertyContextAsync>d__6.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.ProjectPropertiesEditor.<>c__DisplayClass0_0.<<-ctor>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Microsoft.VisualStudio.ProjectSystem.VS.Implementation.PropertyPages.Designer.AsyncLoadContent.<>c__DisplayClass0_0.<<Initialize>b__0>d.MoveNext()
```

The property looks like this:
```xml
  <BoolProperty Name="TargetMultipleFrameworks"
                DisplayName="Target multiple frameworks"
                Description="Build this project for multiple target frameworks."
                HelpUrl="https://go.microsoft.com/fwlink/?linkid=2147236"
                Category="General"
                Visible="False">
    <BoolProperty.DataSource>
      <DataSource Persistence="ProjectFileWithInterception"
                  HasConfigurationCondition="False" />
    </BoolProperty.DataSource>
  </BoolProperty>
```